Proofread Logs Bridge API (#3269)
open-telemetry/opentelemetry-specification: specification/logs/bridge-api.md

The Logs Bridge API consist of these main classes:

> * [LoggerProvider](#loggerprovider) is the entry point of the API. It provides access to Loggers.

Reviewer: ```suggestion
* [LoggerProvider](#loggerprovider) is the entry point of the API. It provides access to `Logger`s.
```

the specified value is invalid SHOULD be logged. A library implementing the
OpenTelemetry API may also ignore this name and return a default instance for
all calls, if it does not support "named" functionality (e.g. an implementation
which is not even observability-related). A `LoggerProvider` could also return a
no-op `Logger` here if application owners configure the SDK to suppress telemetry
produced by this library.
- `version` (optional): Specifies the version of the instrumentation scope if
the scope has a version (e.g. a library version). Example value: 1.0.0.
- `schema_url` (optional): Specifies the Schema URL that should be recorded in
the emitted telemetry.
- `include_trace_context` (optional): Specifies whether the Trace Context should

>   automatically be passed on to the `LogRecords` emitted by the `Logger`. This

Reviewer: ```suggestion
  automatically be passed on to the `LogRecord`s emitted by the `Logger`. This
```

approaches.

The Appender implementation will typically acquire a [Logger](#logger) from the
global [LoggerProvider](#loggerprovider) at startup time, then construct

> `LogRecord`s for received from the application and will call

Reviewer 2: Looks like this is missing a word at least. Maybe better as:
```suggestion
a `LogRecord` for each log received from the application, and then call
```

Author: Fixed.
